inc/template-tags.php

```diff
@@ -219,16 +219,8 @@ function twentynineteen_discussion_avatars_list( $comment_authors ) {
 	function twentynineteen_comment_form( $order ) {
 		if ( true === $order || strtolower( $order ) === strtolower( get_option( 'comment_order', 'asc' ) ) ) {
 
-			/* Only show avatar for logged in users */
```

Comment:
Removing the avatar altogether here. It’s only there as a design decision but isn’t very useful.